survey/SHACL_SurveyResultDataOffering.ttl

```diff
@@ -0,0 +1,52 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix surveyonto: <http://semanticweb.org/metadatasurveyontology/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+surveyonto:InstantiatedVirtualResourceShape
+	a sh:NodeShape ; 
+	sh:targetClass surveyonto:InstantiatedVirtualResource ; 
+	sh:property [
+		sh:path surveyonto:survey_start_time ; 
+		sh:datatype xsd:dateTime ;  
+		skos:example "'Date 1'" ; 
+		sh:minCount 1 ; 
+		sh:message "Missing survey start time." ;
```

Comment:
I would write something like "Validation of survey start time failed". Please adapt for all others